sdks/java/extensions/sql/src/main/codegen/config.fmpp

```diff
@@ -42,6 +42,297 @@ data: {
 
       # List of keywords from "keywords" section that are not reserved.
       nonReservedKeywords: [
+        "A"
```

Comment:
Is there a way to explain this list? Or is it just a fairly arbitrary laundry list of reserved words that are somehow too aggressive?

Reply:
I think you'll have to take this up with calcite: https://github.com/apache/calcite/commit/561c6427e3602510d619b45cd7da840773fa0d2c
```
Compatibility warning: If you have created your own sub-parser you must
copy nonReservedKeywords from core config.fmpp to your config.fmpp.
Previously the core non-reserved keywords were in a parser method
CommonNonReservedKeywords(), but that method has been removed.
```